modules/dop/providers/projectpipeline/service_impl.go

```diff
@@ -245,14 +248,37 @@ func (p *ProjectPipelineService) Delete(ctx context.Context, params deftype.Proj
 		return nil, apierrors.ErrDeleteProjectPipeline.InvalidParameter(err)
 	}
 
-	_, source, err := p.getPipelineDefinitionAndSource(params.ID)
+	definition, source, err := p.getPipelineDefinitionAndSource(params.ID)
 	if err != nil {
 		return nil, apierrors.ErrDeleteProjectPipeline.InvalidParameter(err)
 	}
+	if definition.Creator != apis.GetUserID(ctx) {
```

Comment:
If person resigned, this pipeline can never be deleted?

Reply:
For the time being, do permission management later

Reply:
add TODO please